### PR TITLE
Crystal/   punk-update

### PR DESCRIPTION
**Pull request by Agoichi**
## Media

https://github.com/crystallpunk-14/crystall-punk-14/assets/92464780/c0ed4ec2-8cd1-40f7-a536-7728d6ec2dc3

image

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by Deserty0** [comment]
Description is for lohov

File: Resources/Prototypes/_CP14/Entities/Structures/crystal.yml [hunk 1]
```diff
@@ -11,12 +11,23 @@
       - state: big
         map: ["random"]
         shader: unshaded
-    noRot: true
+    noRot: true   
+  - type: InteractionPopup
+    interactSuccessString: shiny
```
Comment (TheShuEd) [inline Resources/Prototypes/_CP14/Entities/Structures/crystal.yml:16]: Need Localization

File: Resources/Prototypes/_CP14/SoundCollections/hit_impacts.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+- type: soundCollection
+  id: crystaldings
```
Comment (TheShuEd) [inline Resources/Prototypes/_CP14/SoundCollections/hit_impacts.yml:3]:
```suggestion
  id: CP14CrystalDings
```

File: Resources/Prototypes/_CP14/Entities/Structures/crystal.yml [hunk 3]
```diff
@@ -11,12 +11,23 @@
       - state: big
         map: ["random"]
         shader: unshaded
-    noRot: true
+    noRot: true   
+  - type: InteractionPopup
+    interactSuccessString: popup-cp14crystal-ding
+    messagePerceivedByOthers: popup-cp14crystal-ding
+    interactSuccessSound:
+      collection: crystaldings
```
Comment (TheShuEd) [inline Resources/Prototypes/_CP14/Entities/Structures/crystal.yml:19]:
```suggestion
      collection: CP14CrystalDings
```

File: Resources/Prototypes/_CP14/Entities/Structures/crystal.yml [hunk 4]
```diff
@@ -11,12 +11,23 @@
...
+      params:
+        variation: 0.03
   - type: Physics
```
Comment (TheShuEd) [inline Resources/Prototypes/_CP14/Entities/Structures/crystal.yml:22]: Some components are redundant after changing the parent

File: Resources/Prototypes/_CP14/Entities/Structures/crystal.yml [hunk 5]
```diff
@@ -11,12 +11,23 @@
       - state: big
         map: ["random"]
```
Comment (TheShuEd) [inline Resources/Prototypes/_CP14/Entities/Structures/crystal.yml:12]: useless, since the random sprite has been removed.
```suggestion
```

**Review by TheShuEd — CHANGES_REQUESTED** [review]

**Comment by Agoichi** [comment]
Done i guess

File: Resources/Prototypes/_CP14/Entities/Structures/crystal.yml (hunk 4 above)
Comment (TheShuEd) [reply Resources/Prototypes/_CP14/Entities/Structures/crystal.yml:21]: !

Comment (Agoichi) [reply Resources/Prototypes/_CP14/Entities/Structures/crystal.yml:21]: You mean Crystaldings?

Comment (Agoichi) [reply Resources/Prototypes/_CP14/Entities/Structures/crystal.yml:21]: image
We already changed that

**Review by TheShuEd — APPROVED** [review]

File: Resources/Prototypes/_CP14/Entities/Structures/crystal.yml (hunk 4 above)
Comment (TheShuEd) [reply Resources/Prototypes/_CP14/Entities/Structures/crystal.yml:21]: nope, i about static body

**Pull request merged**